### PR TITLE
(webpack) giraffe's webpack.config.js updated for scss files

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
The scss files were'nt working properly because of some mis-config in the webpack, this PR attempts too fix it.

### CODE REVIEW

**Comment by TCL735** [comment]
Currently, I think a diff will be created when `yarn run build` is run, such as when publishing a new version. Or when running it locally with monitor-ci. The diff could cause confusion.

There are a couple of things we need to do:

- We need to get ahead of it and just commit the current outstanding diff now.

- Also, there should be documentation added somewhere to clarify that the .d.ts files will change any time CSS is changed or added for any component, and `yarn run build` is executed. In any future pull requests, the diffs created from doing so need to be committed.

**Review by TCL735 — CHANGES_REQUESTED** [review]
There are uncommitted diffs for the .d.ts files

**Comment by ChitlangeSahas** [comment]
We had to remove the Prettier Plugin because it doesn't support the `prettierignore` files at the moment. I tested it with the start command and it successfully ignored prettifying the generated files.

File: stories/package.json [hunk 1]
```diff
@@ -36,7 +36,6 @@
     "intl-dateformat": "^0.1.1",
     "memoize-one": "^5.0.2",
     "prettier": "^1.19.1",
-    "prettier-webpack-plugin": "^1.2.0",
```
Comment (TCL735) [inline stories/package.json:39]: This should be removed from yarn.lock as well with a `yarn install` and then committed

**Review by TCL735 — CHANGES_REQUESTED** [review]

File: stories/package.json (hunk 1 above)
Comment (ChitlangeSahas) [reply stories/package.json:39]: Absolutely, my bad I should have removed it.

**Review by TCL735 — APPROVED** [review]

**Pull request merged**